src/Akka.Streams.ScanMemory/Program.cs

```diff
@@ -14,6 +14,7 @@ static async Task Main(string[] args)
         {
             var system = ActorSystem.Create("MySystem");
             var streamBits = Source.ActorRef<int>(1000, OverflowStrategy.DropHead)
+                .Select(x => ThreadLocalRandom.Current.Next(1, 10000))
```

Comment:
Generate a spread of random numbers - should force the `SortedSet<int>` to grow in size over time, but it'll remain bounded in size.